app/views/work/edit.html.slim

```diff
@@ -239,3 +242,30 @@
         };
       });
     });
+
+-content_for :javascript
+  javascript:
+    $(document).ready(function() {
+      const collectionSelect = $('#work_collection_id');
+      const documentSetsDiv = $('#document_sets');
+
+      function updateDocumentSetOptions(selectedId) {
+        $.ajax({
+          url: "#{work_document_sets_select_path}",
+          method: 'GET',
+          data: { collection_id: selectedId },
+          success: function(response) {
+            documentSetsDiv.html(response);
```

Comment:
We fetch to work_document_sets_select_path, it's response is an html partial. In which case we will be using it's response to replace the contents of our html target